### DISCUSSION TITLE
Update to 9.0.1 (41929) Broke browsing into S3 Buckets [Beta Builds channel]

### DISCUSSION THREAD

**Discussion by lsascha** [iterate-ch/cyberduck#16164] [Q&A]
I have 3 S3 Server with multiple buckets.

Since the last update, every time i try to browse into one of the buckets, i get the following error message:
![image](https://github.com/user-attachments/assets/13097677-15f7-480d-b8ce-e4e9dcd1e67e)

On all 3 Servers it is the same error, and i think it is relatively unlikely that all 3 of them have the same issue at the same time.
And uploading it through other ways still works.

To reproduce:
- Connect to a S3 compatible server.
- double click on a bucket
- get the error message.

Expected behavior:
- Connect to a S3 compatible server.
- double click on a bucket
- Get the list of all folders in that bucket

Desktop (please complete the following information):
 - OS: Windows 11

**Comment by dkocher** [comment]
This is caused by the changeset in #16079. Please switch to use the _S3 (Deprecated path style requests)_ connection profile from _Preferences → Profiles_.

**Comment by lsascha** [comment]
Then please make the error message more meaningful.

And "deprecated" means it will be removed in the future?

Please don't do that, since that is (i think) the only way that most S3 compatible servers work, like minio and probably others.
(correct me if i am wrong).

**Reply by AliveDevil** [reply]
Deprecated, because AWS doesn't support it anymore.

Most S3 compatible servers also support the DNS-style requests (which are used by AWS) - as long as they are set up correctly (i.e. domain name, and CNames for buckets), e.g. for MinIO this is referred to as Virtual Host-style[^1] requests.

[^1]: https://min.io/docs/minio/linux/reference/minio-server/settings/core.html#envvar.MINIO_DOMAIN

**Comment by dkocher** [comment]
We have improved the documentation [^1].

[^1]: iterate-ch/docs#555

**Comment by lsascha** [comment]
thanks. Another thing though.

Shouldn't the used Profile for a favorite be staying as it is?

As it is now, if someone used the the Profile in a favorite to connect to a server, it will suddenly stop working without much explanation.

Better would be:
- Rename the existing Profile, so existing favorites for connections continue to work. You can name it (deprecated) then.
- Create a new Profile that does not support the path style requests.

This way, old favorites will continue to work, which results in a better user experience.
